lmnet/lmnet/common.py

```diff
@@ -52,5 +52,5 @@ class Tasks(Enum):
 
 def get_color_map(length):
     # This function generate arbitrary length color map.
-    color_map = COLOR_MAP * math.ceil(length / len(COLOR_MAP))
+    color_map = COLOR_MAP * int(math.ceil(float(length) / len(COLOR_MAP)))
```

Comment:
You can use __future__ division instead of float

Reply:
https://www.python.org/dev/peps/pep-0238/#the-future-division-statement

Reply:
Thanks, from `__future__` import division with commit 70937f8.